lib/changed_rpm_files_helper.rb

```diff
@@ -93,6 +93,10 @@ def parse_stat_line(line)
     ]
   end
 
+  def get_link_target(system, link)
+    system.run_command("readlink", link, stdout: :capture).strip
```

Comment:
Can you please add `readlink` to list of required commands on the man page https://github.com/SUSE/machinery/blob/master/man/machinery-inspect.1.md ?